build/Makefile

```diff
@@ -62,8 +66,25 @@ NPM_PKGS_DIR:=node_modules
 $(NPM_PKGS_DIR):
 	$(MAKE) -C $(OSCAL_CORE_DIR) dependencies
 
+OSCAL_CLI_VERSION=`curl -s https://api.github.com/repos/usnistgov/oscal-cli/releases/latest | jq -r '.name[1:]'`
+OSCAL_CLI_BIN:=oscal-cli
+OSCAL_CLI_INSTALL_URL:=https://repo1.maven.org/maven2/gov/nist/secauto/oscal/tools/oscal-cli/cli-core/$(OSCAL_CLI_VERSION)/cli-core-$(OSCAL_CLI_VERSION)-oscal-cli.zip
+OSCAL_CLI_INSTALL_PATH := $(shell which oscal-cli 2>/dev/null)
+ifeq ($(OSCAL_CLI_INSTALL_PATH),)
```

Comment:
`oscal-cli` gets installed anyways in the build/oscal-cli dir despite of an existing installation, but then the already installed version is being used, making the installation in the build/oscal-cli useless. The Makefile will work for the pipeline.

I suggest changing the logic slightly to:
```
OSCAL_CLI_INSTALL_PATH := ./oscal-cli/
$(OSCAL_CLI_INSTALL_PATH):	
	@echo "Downloading OSCAL CLI version $(OSCAL_CLI_VERSION)..."; \
	mkdir -p $(OSCAL_CLI_INSTALL_PATH); \
	curl $(CURL_INSTALL_OPTS) -o $(OSCAL_CLI_INSTALL_PATH)/oscal-cli.zip $(OSCAL_CLI_INSTALL_URL); \
	unzip -o $(OSCAL_CLI_INSTALL_PATH)/oscal-cli.zip -d $(OSCAL_CLI_INSTALL_PATH); \
	chmod +x $(OSCAL_CLI_INSTALL_PATH)/bin/$(OSCAL_CLI_BIN)
else
OSCAL_CLI_INSTALL_PATH := $(shell dirname $$(dirname $$(which oscal-cli)))
endif
```